src/PrestaShopBundle/Resources/views/Admin/Improve/International/Currency/Blocks/form.html.twig

```diff
@@ -42,7 +42,11 @@
         'list.example': 'Example'|trans({}, 'Admin.Global'),
         'list.edit': 'Edit symbol / format'|trans({}, 'Admin.International.Feature'),
         'list.reset': 'Reset settings'|trans({}, 'Admin.Actions'),
-        'list.example.format': 'Price: %price% (Special discount %discount%)'|trans({}, 'Admin.International.Feature')
+        'list.example.format': 'Price: %price% (Special discount %discount%)'|trans({}, 'Admin.International.Feature'),
+        'modal.confirm.title': 'Are you sure you want to restore default settings?'|trans({}, 'Admin.International.Feature'),
```

Comment:
On second thought, maybe `modal.confirm.*` is a bit generic, especially for `modal.confirm.body` Maybe we should use something a bit more related to context, in case some day another modal is added Like `modal.restore.title` and `modal.restore.body` As for apply and cancel since they are quite generic it doesn't shock me that much

Reply:
done